src/components/modal/index.web.tsx

```diff
@@ -0,0 +1,110 @@
+// import _ from 'lodash';
+import React, {Component} from 'react';
+import {createPortal} from 'react-dom';
+import {ModalProps as RNModalProps, GestureResponderEvent, KeyboardAvoidingViewProps} from 'react-native';
+import {BlurViewPackage} from '../../optionalDependencies';
+import {asBaseComponent} from '../../commons/new';
+import TopBar, {ModalTopBarProps} from './TopBar';
+import View from '../../components/view';
+
+const BlurView = BlurViewPackage?.BlurView;
+
+export {ModalTopBarProps};
+export interface ModalProps extends RNModalProps {
```

Comment:
I don't think it's a good idea to copy all the props here, because if we'll change something in the original file we'll have to remember to add it here as well and probably forget. 
I suggest creating a types file, move all the types to this file and import it from there in the component